src/Resources/doc/configuration_reference.md

```diff
@@ -8,9 +8,9 @@ csa_guzzle:
 
         # The maximum size of the body which should be stored in the profiler (in bytes)
         max_body_size:        65536 # Example: 65536
-
-    # Whether or not to enable the logger
-    logger:               false # Example: %kernel.debug%
+    logger:
+        enabled:              false
+        service:              null
```

Comment:
Nope. Not needed. That's just to be consistent with the rest of the documentation, which uses `~` to represent `null`.